### ISSUE TITLE
Please add support for H607C Govee Floor Lamp 2

### ISSUE THREAD

**Issue by beslergr** [wez/govee2mqtt#178]
Getting an "unknown device type" when trying to connect the new Govee Floor Lamp 2. Can this be added?

**Comment by wez** [comment]
In general, we don't need to add explicit support for lights because the govee api is very generic.  If something isn't working, we need more data and logs.

**Comment by agoode** [comment]
I have one of these devices. It's a multisegment device, but only basic controls are exposed to Home Assistant.

I've connected with the LAN, the IOT, and the API. Here is a log snippet:

```
[2024-05-08T19:20:05 INFO  govee::commands::serve] Starting service. version 2024.04.29-30cf7732
[2024-05-08T19:20:05 INFO  govee::commands::serve] Querying platform API for device list
[2024-05-08T19:20:05 INFO  govee::commands::serve] Querying undocumented API for device + room list
[2024-05-08T19:20:05 INFO  govee::service::iot] Connected to IoT: CONNACK code 0: Connection Accepted.
[2024-05-08T19:20:05 INFO  govee::commands::serve] Starting LAN discovery
[2024-05-08T19:20:05 INFO  govee::commands::serve] Waiting 10 seconds for LAN API discovery
[2024-05-08T19:20:05 INFO  govee::service::iot] IoT (re)connected with status CONNACK code 0: Connection Accepted.
[2024-05-08T19:20:15 INFO  govee::commands::serve] Devices returned from Govee's APIs
[2024-05-08T19:20:15 INFO  govee::commands::serve] H607C_ABA3 (52:52:C7:01:D5:DF:AB:A3 H607C)
[2024-05-08T19:20:15 INFO  govee::commands::serve]   Undoc: room=None supports_iot=true ble_only=false
[2024-05-08T19:20:15 WARN  govee::commands::serve]   Unknown device type. Cannot map to Home Assistant.
[2024-05-08T19:20:15 INFO  govee::commands::serve] 
```

Let me know what more information you need. Thanks!

**Comment by agoode** [comment]
[log.txt](https://github.com/wez/govee2mqtt/files/15256952/log.txt)

**Comment by agoode** [comment]
I can open a new issue if that is helpful.

**Comment by wez** [comment]
FWIW, from those logs, it looks like we're waiting on govee to make this device show up in their platform api before we can do anything with it.

**Comment by beslergr** [comment]
Apologies on the radio silence here. Had to get a replacement lamp from Govee, and then had some major client work to take care of. @wez let me know if you need anything from my end, but thanks @agoode for sending the logs! Fingers crossed this gets added to the API soon.

**Comment by Raul-7-7** [comment]
@tweaksalot I just got the device, I can change the colour/brightness and turn it off/on. But the scenes are missing :(

**Comment by aaltjoe** [comment]
@Raul-7-7 Were you able to get the H607C listed in the "devices" request response?

**Comment by ArthurGarnier** [comment]
Hi, 
Just got the H607C, everything is OK in HA, can control light and use Scenes.
![image](https://github.com/user-attachments/assets/55148f20-a80e-4fe1-b414-b41cc6a62844)
